src/testing/repository_manager.go

```diff
@@ -38,6 +38,12 @@ func (f *FakeRepositoryManager) Get(ctx context.Context, id int64) (*models.Repo
 	return args.Get(0).(*models.RepoRecord), args.Error(1)
 }
 
+// GetByName ...
+func (f *FakeRepositoryManager) GetByName(ctx context.Context, name string) (*models.RepoRecord, error) {
+	args := f.Called()
+	return args.Get(0).(*models.RepoRecord), args.Error(1)
```

Comment:
For such a simple return statement, you need to confirm no error case happened in your case otherwise `args.Get(0).(*models.RepoRecord)` will be panic